crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -358,6 +361,7 @@ impl kitsune_p2p::event::KitsuneP2pEventHandler for HolochainP2pActor {
         }
     }
 
+    #[tracing::instrument(skip(self))]
```

Comment:
```suggestion
    #[tracing::instrument(skip(self), level = "trace")]
```
Can you please set all the actor instruments to trace like this because otherwise the generate debug spans and it creates way to much data when trying to do a flame graph. It also slows down tracing in general as these are hot paths